File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/SplitDocumentResult.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class SplitDocumentResult {
    public Link SourceDocument { get; set; }

    public List<Link> Pages { get; set; }

    public Link ZippedPages { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SplitDocumentResult {\n");
      sb.Append("  SourceDocument: ").Append(SourceDocument).Append("\n");
      sb.Append("  Pages: ").Append(Pages).Append("\n");
      sb.Append("  ZippedPages: ").Append(ZippedPages).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/SplitDocumentResult.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class SplitDocumentResult {
  private Link SourceDocument = null;
  private List<Link> Pages = new ArrayList<Link>();
  private Link ZippedPages = null;
  /**
	 * getSourceDocument
	 * Gets Link
	 * @return SourceDocument
	 */
  public Link getSourceDocument() {
    return SourceDocument;
  }

	/**
	 * setSourceDocument
	 * Sets Link
	 * @param SourceDocument Link
	 */
  public void setSourceDocument(Link SourceDocument) {
    this.SourceDocument = SourceDocument;
  }

  /**
	 * getPages
	 * Gets List<Link>
	 * @return Pages
	 */
  public List<Link> getPages() {
    return Pages;
  }

	/**
	 * setPages
	 * Sets List<Link>
	 * @param Pages List<Link>
	 */
  public void setPages(List<Link> Pages) {
    this.Pages = Pages;
  }

  /**
	 * getZippedPages
	 * Gets Link
	 * @return ZippedPages
	 */
  public Link getZippedPages() {
    return ZippedPages;
  }

	/**
	 * setZippedPages
	 * Sets Link
	 * @param ZippedPages Link
	 */
  public void setZippedPages(Link ZippedPages) {
    this.ZippedPages = ZippedPages;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SplitDocumentResult {\n");
    sb.append("  SourceDocument: ").append(SourceDocument).append("\n");
    sb.append("  Pages: ").append(Pages).append("\n");
    sb.append("  ZippedPages: ").append(ZippedPages).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

